RHDHBUGS-2625: Add procedure to configure Orchestrator to connect to existing PostgreSQL infrastructure (#2199)
redhat-developer/red-hat-developers-documentation-rhdh: modules/shared/snip-create-kubernetes-service-for-external-postgresql.adoc

:_mod-docs-content-type: SNIPPET

> 

Reviewer: ```suggestion
```

Reviewer: The include results in too many blank lines, remove one line here, or before the include statement.

<img width="692" height="342" alt="Image" src="https://github.com/user-attachments/assets/10acf6e4-4ee8-4738-bbe7-542b85f443cb" />

> . Create a Kubernetes Service that points to your external PostgreSQL database:

Reviewer: ```suggestion
. Create a Kubernetes service that points to your external PostgreSQL database:
```

+
[source,yaml,subs="+attributes,+quotes"]
----
apiVersion: v1
kind: Service
metadata:
  name: external-postgresql-service
spec:
  type: ExternalName
  externalName: __<your-external-db-hostname>__
  ports:
    - port: 5432
      targetPort: 5432
      protocol: TCP
----
+
Where:

> `external-postgresql-service`:: Name of the Service to reference in plugin configurations.

Reviewer: ```suggestion
`external-postgresql-service`:: Name of the service to reference in plugin configurations.
```

`ExternalName`:: Service type that creates a CNAME record to the external database hostname.
`__<your-external-db-hostname>__`:: FQDN of your external PostgreSQL server, for example, `postgres.example.com`.
+
[NOTE]
====

> If your external database is outside the cluster or uses an IP address instead of a hostname, create a Service with Endpoints:

Reviewer: ```suggestion
If your external database is outside the cluster or uses an IP address instead of a host name, create a service with endpoints:
```
